core/blockchain.go

```diff
@@ -180,7 +180,7 @@ func NewBlockChain(db ethdb.Database, cacheConfig *CacheConfig, chainConfig *par
 	if cacheConfig == nil {
 		cacheConfig = &CacheConfig{
 			TrieNodeLimit: 256 * 1024 * 1024,
-			TrieTimeLimit: 5 * time.Minute,
+			TrieTimeLimit: 2 * time.Minute,
```

Comment:
why decreased to 2 minutes?
This means we flush to disk more frequently than before, thus less aggressive pruning.
Why not do more aggressive pruning with larger trie time limit since we have less number of transactions and states history changes less?